code/modules/projectiles/guns/energy.dm

```diff
@@ -65,11 +65,16 @@
 	readout += "Standard models of this projectile weapon have [span_warning("[ammo_type.len]")] mode\s"
 	readout += "Our heroic interns have shown that one can theoretically stay standing after..."
 	for(var/obj/item/ammo_casing/energy/for_ammo in ammo_type)
-		exam_proj = for_ammo.loaded_projectile
+		if(!for_ammo.projectile_type)
+			continue
+		exam_proj = new for_ammo.projectile_type() // Initial loaded projectile cannot be used because its initial value is null
```

Comment:
Make a global list of singletons per type instead.
Should be cleaner that creating and destroying one each time.